README.md

```diff
@@ -1,28 +1,28 @@
 [![Build Status](https://travis-ci.org/OCA/connector-ecommerce.svg?branch=10.0)](https://travis-ci.org/OCA/connector-ecommerce)
-[![Coverage Status](https://coveralls.io/repos/OCA/connector-ecommerce/badge.png?branch=10.0)](https://coveralls.io/r/OCA/connector-ecommerce?branch=7.0)
+[![Coverage Status](https://coveralls.io/repos/OCA/connector-ecommerce/badge.png?branch=10.0)](https://coveralls.io/github/OCA/connector-ecommerce?branch=10.0)
 
 Odoo Connector Ecommerce Modules
 ================================
 
-*OpenERP Connector is a powerful framework to develop any kind of bi-directional connector between OpenERP (Open Source ERP) and any other software or service.*
+*Odoo Connector is a powerful framework to develop any kind of bi-directional connector between Odoo (Open Source ERP formerly OpenERP) and any other software or service.*
 
-This OpenERP add-on has a modular and generic core, with the ability to be extended with additional modules for new features or customizations.
+This Odoo add-on has a modular and generic core, with the ability to be extended with additional modules for new features or customizations.
 
 This project contains modules extending the connector for projects dealing with E-Commerce.
 
-The team for the community is here: https://launchpad.net/~openerp-connector-community
-Please join and subscribe to the mailing list.
+Feel free to subscribe on the mailing list (its name is 'Connectors'):
+https://odoo-community.org/groups
 
 The project for the **Connector** framework is: https://github.com/OCA/connector
 
 Documentation:
-http://www.openerp-connector.com
+http://www.odoo-connector.com
 
 [//]: # (addons)
 Unported addons
 ---------------
 addon | version | summary
 --- | --- | ---
-[connector_ecommerce](connector_ecommerce/) | 9.0.1.0.0 (unported) | Connector for E-Commerce
+[connector_ecommerce](connector_ecommerce/) | 10.0.1.0.0 (unported) | Connector for E-Commerce
```

Comment:
Don't modify this or there might be conflicts. And please amend current commit for fixing it.